mailpoet/tests/unit/Newsletter/Statistics/Export/StatisticsExporterTest.php

```
@@ -0,0 +1,162 @@
+  public function testBuildAggregateRowMatchesHeaderColumnCount() {
+    $newsletter = $this->createNewsletter(1, 's', null, null);
+    $stats = $this->createStats(0, 0, 0, 0, 0, 0, null);
+    $exporter = $this->createExporter($newsletter, $stats);
+
+    $headers = $exporter->getAggregateHeaders();
+    $row = $exporter->buildAggregateRow($newsletter, $stats);
+
+    verify(count($row))->equals(count($headers));
+  }
+
+  private function createExporter(NewsletterEntity $newsletter, NewsletterStatistics $stats): StatisticsExporter {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$newsletter'. (undefined)

(UnusedFormalParameter)